system_tests/monitoring.py

```diff
@@ -0,0 +1,133 @@
+            self.assertTrue(label.description)
+
+    def test_list_metric_descriptors(self):
+        client = monitoring.Client()
+
+        descriptor = None
+        for item in client.list_metric_descriptors():
+            if item.type == METRIC_TYPE:
+                descriptor = item
+
+        self.assertIsNotNone(descriptor)
+
+        expected_name = 'projects/{project}/{what}/{type}'.format(
+            project=client.project,
+            what='metricDescriptors',
+            type=METRIC_TYPE,
+        )
+        self.assertEqual(descriptor.name, expected_name)
+        self.assertEqual(descriptor.type, METRIC_TYPE)
+        self.assertEqual(descriptor.metric_kind, METRIC_KIND)
+        self.assertEqual(descriptor.value_type, VALUE_TYPE)
+        self.assertTrue(descriptor.description)
+
+        self.assertTrue(descriptor.labels)
+        for label in descriptor.labels:
+            self.assertTrue(label.key)
+            self.assertTrue(label.value_type)
+            self.assertTrue(label.description)
+
+    def test_fetch_resource_descriptor(self):
+        client = monitoring.Client()
+        descriptor = client.fetch_resource_descriptor(RESOURCE_TYPE)
+
+        expected_name = 'projects/{project}/{what}/{type}'.format(
+            project=client.project,
+            what='monitoredResourceDescriptors',
+            type=RESOURCE_TYPE,
+        )
+        self.assertEqual(descriptor.name, expected_name)
+        self.assertEqual(descriptor.type, RESOURCE_TYPE)
+        self.assertTrue(descriptor.display_name)
+        self.assertTrue(descriptor.description)
+
+        self.assertTrue(descriptor.labels)
+        for label in descriptor.labels:
+            self.assertTrue(label.key)
+            self.assertTrue(label.value_type)
+            self.assertTrue(label.description)
+
+    def test_list_resource_descriptors(self):
+        client = monitoring.Client()
+
+        descriptor = None
+        for item in client.list_resource_descriptors():
+            if item.type == RESOURCE_TYPE:
+                descriptor = item
+
+        self.assertIsNotNone(descriptor)
+
+        expected_name = 'projects/{project}/{what}/{type}'.format(
+            project=client.project,
+            what='monitoredResourceDescriptors',
+            type=RESOURCE_TYPE,
+        )
+        self.assertEqual(descriptor.name, expected_name)
+        self.assertEqual(descriptor.type, RESOURCE_TYPE)
+        self.assertTrue(descriptor.display_name)
+        self.assertTrue(descriptor.description)
+
+        self.assertTrue(descriptor.labels)
+        for label in descriptor.labels:
+            self.assertTrue(label.key)
+            self.assertTrue(label.value_type)
+            self.assertTrue(label.description)
+
+    def test_query(self):
+        client = monitoring.Client()
+        query = client.query(METRIC_TYPE, hours=1)
+        # There may be no data, but we can ask anyway.
+        for _ in query:
+            pass
```

Comment:
I added the comment "Not necessarily reached" to the `pass` statement.

I don't know of a good metric to look for in empty projects. Creating, say, a pub-sub topic in the system test wouldn't help, because of latency.

I don't want to require manual setup like for the Datastore system tests. It wouldn't help us anyway with the write side of the API when we get to that.

@jgeewax told me earlier that he was okay with not blocking this pull request on system tests.